tests/gptq/test_quantization.py

```diff
@@ -142,6 +144,7 @@ def test_serialization(self):
         with tempfile.TemporaryDirectory() as tmpdirname:
             self.quantizer.save(self.quantized_model, tmpdirname)
             self.quantized_model.config.save_pretrained(tmpdirname)
+            print(self.quantized_model.config)
```

Comment:
to remove

Reply:
I happen to not have maintainer rights anymore so can't push myself unfortunately

Reply:
mb thx !